src/main/java/com/minipulse/db/MiniPulseDB.java

```diff
@@ -60,31 +60,144 @@ String getPollIdForUserByTitle(String user, String pollTitle)
                         return null;
                 }
 
-            void savePollAndOverwriteQuestions(Poll poll);

```

Comment:
Create a new class src/main/java/com/minipulse/db/MiniPulseDBImpl.java and move all the database related code  to that class. 

MIniPulseDBImpl should implement MiniPulseDB and all its methods

Restore the original contents of interface MiniPulseDB.java